libs/vscode-ui/components/src/lib/autocomplete/autocomplete.component.html

```diff
@@ -2,7 +2,7 @@
   {{ field.description }}
```

Comment:
These form fields didn't have any awareness of the Validators set on the parent form group, so I connected them up with ControlContainer.